queries.sql

```diff
@@ -1,4 +1,8 @@
 UPDATE teams 
 SET abbreviation = WAS
   WHERE abbreviation
-IS WHS;
\ No newline at end of file
+IS WHS;
+
+SELECT *
+FROM teams
+WHERE league LIKE 'NL';
```

Comment:
You don't need `LIKE` since you are not using a wild card.

`SELECT * FROM teams WHERE league = 'NL';`